plugins/uartdmx/UartDmxThread.h

```diff
@@ -49,10 +49,15 @@ class UartDmxThread : public ola::thread::Thread {
   DmxBuffer m_buffer;
   ola::thread::Mutex m_term_mutex;
   ola::thread::Mutex m_buffer_mutex;
+  unsigned int m_frameTime;
```

Comment:
Could we stick to snake_case here to match the other variables:
```suggestion
  unsigned int m_frame_time;
```

Reply:
Done